### ISSUE TITLE
Test Core_Arith_Regression24163, test_for_ties_to_even fails on RISC-V

### ISSUE THREAD

**Issue by asmorkalov** [opencv/opencv#24746]
### System Information

Linux, RISC-V RVV

### Detailed description

PR: https://github.com/opencv/opencv/pull/24271

Example:
```
[ RUN      ] Core_Arith_Regression24163.test_for_ties_to_even/240, where GetParam() = (CV_32S, 3, -2, -1)
/home/ci/opencv/modules/core/test/test_operations.cpp:1605: Failure
Expected equality of these values:
  0
  cvtest::norm(expected, result, NORM_INF)
    Which is: 1
result=
[-1, -1, -2;
 -2, -1, -1;
 -2, -2, -2]
expected=
[-2, -2, -2;
 -2, -2, -2;
 -2, -2, -2]
[  FAILED  ] Core_Arith_Regression24163.test_for_ties_to_even/240, where GetParam() = (CV_32S, 3, -2, -1) (2 ms)
```

### Steps to reproduce

Enable the test and re-start CI

### Issue submission checklist

- [X] I report the issue, it's not a question
- [X] I checked the problem with documentation, FAQ, open issues, forum.opencv.org, Stack Overflow, etc and have not found any solution
- [X] I updated to the latest OpenCV version and the issue is still there
- [X] There is reproducer code and related data files (videos, images, onnx, etc)

**Comment by asmorkalov** [comment]
@mshabunin @hanliutong Is the issue still relevant?

**Comment by mshabunin** [comment]
@asmorkalov , yes, the test is still disabled for RISC-V platforms.

**Comment by hanliutong** [comment]
@asmorkalov, fix by PR #25586 just now.
